hyperspy/_components/eels_cl_edge.py

```diff
@@ -69,27 +69,45 @@ class EELSCLEdge(Component):
         favourable cases is proportional to the number of atoms of
         the element. It is a component.Parameter instance.
         It is fixed by default.
-    fine_structure_coeff : Parameter
-        The coefficients of the spline that fits the fine structure.
-        Fix this parameter to fix the fine structure. It is a
-        component.Parameter instance.
     effective_angle : Parameter
         The effective collection semi-angle. It is automatically
         calculated by set_microscope_parameters. It is a
         component.Parameter instance. It is fixed by default.
-    fine_structure_smoothing : float between 0 and 1
+    fine_structure_active : bool, default False
+        Activates/deactivates the fine structure features. When active,
+        the fine structure region model is not defined by the simulated
```

Comment:
What is a "fine structure region model"? A model for the fine structure region, or a region, where the fine structure is modeled, ...?

Reply:
Fixed, thanks.